apps/src/signUpFlow/FinishStudentAccount.tsx

```diff
@@ -154,11 +154,11 @@ const FinishStudentAccount: React.FunctionComponent<{
         />
       </fieldset>
       <div className={style.finishSignUpButtonContainer}>
-        <Button
+        <LinkButton
           className={style.finishSignUpButton}
           color={buttonColors.purple}
           type="primary"
-          onClick={() => console.log('FINISH SIGN UP')}
+          href="#"
```

Comment:
Is this supposed to link somewhere too?

Reply:
Not yet since this will be where we create the user object then send the user to the homepage.

Reply:
Got it, sounds good!